src/core/theme/SuomifiTheme/shadows.ts

```diff
@@ -10,4 +10,5 @@ export const shadows = (colors: ColorDesignTokens) => ({
   actionElementBoxShadow: `0 1px 2px 0 ${alphaHex(0.1)(
     colors.brandBase,
   )} inset`,
+  notificationBoxShadow: `0px 4px 8px 0px ${alphaHex(0.14)(colors.blackBase)}`,
```

Comment:
Could we try to figure out a little more generic name for the this shadow to be more consistent with other shadows?